stl/inc/flat_map

```diff
@@ -203,6 +216,54 @@ struct _Pairing_iterator_provider {
             return _Const_iterator{_Key_it, _Mapped_it};
         }
 
+        const _KeyIter& _Key_iterator() const noexcept {
+            return _Key_it;
+        }
+
+        const _MappedIter& _Mapped_iterator() const noexcept {
+            return _Mapped_it;
+        }
+
+        using _Prevent_inheriting_unwrap = _Iterator;
+
+        friend void _Verify_range(const _Iterator& _First, const _Iterator& _Last) noexcept {
+            if constexpr (_Range_verifiable_v<_KeyIter>) {
+                _Verify_range(_First._Key_it, _Last._Key_it); // intentional ADL
+            }
+            if constexpr (_Range_verifiable_v<_MappedIter>) {
+                _Verify_range(_First._Mapped_it, _Last._Mapped_it); // intentional ADL
+            }

```

Comment:
Style nitpick: Should have a newline between non-chained `if` statements. Occurs below.